rs/src/version_mod.rs

```diff
@@ -1,22 +1,38 @@
+use crate::git;
+use anyhow::{Context, Result};
 use std::fs;
 use std::path::Path;
-use anyhow::{Result, Context};
 
-//const AT_VERSION_FILE: &str = ".at-version";
-const AT_VERSION_FILE: &str = "RELEASE";
+const AT_VERSION_FILE: &str = ".at-version";
 
 /// Read content from specified at-version file.
 /// If the path is None, it defaults to ".at-version".
 fn read_at_version() -> Result<String> {
-    let version = fs::read_to_string(Path::new(AT_VERSION_FILE))
-        .context(format!("Failed to read from version file: {}", AT_VERSION_FILE))?;
-    Ok(version)
+    let version = fs::read_to_string(Path::new(AT_VERSION_FILE)).context(format!(
+        "Failed to read from version file: {}",
+        AT_VERSION_FILE
+    ))?;
+    Ok(version.trim().to_string())
 }
 
-/// Get current version
-pub fn get() -> Result<String> {
-    let version = read_at_version()
-        .context("Failed to read the version file")?;
-    // Here we should do some Git checks...
+/// Get current version, optionally including git information
+///
+/// If git_info is true, appends branch, hash and dirty status in the format:
+/// version+branch-hash[-dirty]
+pub fn get(git_info: bool) -> Result<String> {
```

Comment:
this should take the path to the project root as an argument, or take a an already "discovered" repo struct